### PR TITLE
Introduce Azure Key Vault background jobs

### PR DESCRIPTION
**Pull request by tomkerkhove**
Introduce Azure Key Vault background jobs which was part of Arcus Web API.

- [x] Create new Service Bus namespace for Arcus Jobs instead of Web API

Relates to https://github.com/arcus-azure/arcus.backgroundjobs/issues/14
Relates to https://github.com/arcus-azure/arcus.webapi/pull/113

### CODE REVIEW

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>Azure;WebAPI;App Services;Web App;Web;API</PackageTags>
+    <Description>Provides capabilities for running background jobs to automate workflows such as updating cached Azure Key Value secrets.</Description>
+    <Copyright>Copyright (c) Arcus</Copyright>
+    <PackageLicenseUrl>https://github.com/arcus-azure/arcus.webapi/blob/master/LICENSE</PackageLicenseUrl>
```
Comment (stijnmoreels) [inline src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: webapi > backgroundjobs?

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <RepositoryType>Git</RepositoryType>
+    <PackageTags>Azure;WebAPI;App Services;Web App;Web;API</PackageTags>
```
Comment (stijnmoreels) [inline src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: Maybe also other tags than the web API could be useful?

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <Description>Provides capabilities for running background jobs to automate workflows such as updating cached Azure Key Value secrets.</Description>
+    <Copyright>Copyright (c) Arcus</Copyright>
+    <PackageLicenseUrl>https://github.com/arcus-azure/arcus.webapi/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/arcus-azure/arcus.webapi</PackageProjectUrl>
```
Comment (stijnmoreels) [inline src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: webapi > backgroundjobs?
```suggestion
<PackageProjectUrl>https://github.com/arcus-azure/arcus.backgroundjobs</PackageProjectUrl>
```

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj (hunk 1 above)
Comment (tomkerkhove) [reply src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: Well spotted!

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
```
Comment (stijnmoreels) [inline src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: webapi > backgroundjobs?
```suggestion
<RepositoryUrl>https://github.com/arcus-azure/arcus.backgroundjobs</RepositoryUrl>
```

File: src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj (hunk 1 above)
Comment (stijnmoreels) [reply src/Arcus.BackgroundJobs.KeyVault/Arcus.BackgroundJobs.KeyVault.csproj]: 🔍 🕵

**Pull request merged**